src/PrestaShopBundle/Resources/config/services/bundle/repository.yml

```diff
@@ -134,6 +134,11 @@ services:
     arguments:
       - PrestaShopBundle\Entity\AttributeGroup
 
+  PrestaShopBundle\Entity\Repository\AttributeRepository:
+    factory: [ '@doctrine.orm.default_entity_manager', getRepository ]
+    arguments:
+      - PrestaShopBundle\Entity\Attribute
+
```

Comment:
Can you please put this new service declaration after `prestashop.core.admin.attribute_group.repository` The idea is to keep the alias close to its original service